iridauploader/gui/config.py

```diff
@@ -68,13 +70,20 @@ def __init__(self, parent=None):
         self._get_settings_from_file()
         self._contact_irida()
 
+        # Error message popup
+        self._error_message_box = QtWidgets.QMessageBox(parent=self)
+        self._error_message_box.setText("Config file was invalid. Please check your file and try again.")
```

Comment:
Do we have a read the docs for the irida-uploader with a page on the config file showing the format that we could link to?

Reply:
Added.